README.md

```diff
@@ -231,7 +231,7 @@ treated_rep2,/path/to/reads/treat2_1.fastq,/path/to/reads/treat2_2.fastq,treated
 treated_rep3,/path/to/reads/treat3_1.fastq,/path/to/reads/treat3_2.fastq,treated,C
```

Comment:
I think we need an extra comma here, do we?
```
mock_rep1,/path/to/reads/mock1_1.fastq,/path/to/reads/mock1_2.fastq,mock,A,
```
So that strandness is set empty

Reply:
Ah, also for the single-end above